### PR TITLE
Add trip fare

### PR DESCRIPTION
**Pull request by sijintv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,6 +138,35 @@ Returns route map details
...
+      ```javascript
+    {
+        'adults' : <noOfAdults>,
+        'destination' : <destination>,
+        'serviceType' : <serviceType>,
+        'source' : <source>
+      }
+      ```
+      Service types can be either
+  * ordinary
```
Comment (tachyons) [inline README.md:161]: Types are camelcase , I converted to snake case to follow ruby convention

Comment (tvsijin) [inline README.md]: actually there arent CamelCase .. they arent variable either <actually forgot to add spaces> ... any way snake_case is here

**Comment by sijintv** [comment]
snake case added

File: README.md (hunk 1 above)
Comment (tachyons) [inline README.md:161]: My bad , I meant service types returned by BMTC api is in in camel case . But value we are passing for fare is just an identifier and case doesn't matter there !

File: README.md [hunk 2]
```diff
@@ -138,6 +138,35 @@ Returns route map details
...
+  * vayu_vajra
+  * atal_sarige
+  * nice_service
+  * bengaluru_darshini
+
+
 # Implementations
  * https://github.com/tachyons/bmtc
  * https://github.com/tvsijin/bmtc-api-php
+ * https://github.com/vineeshnp/bmtc-js-api
```
Comment (tachyons) [inline README.md:172]: Already added 3 days back

Comment (sijintv) [inline README.md:172]: merge it bro.. there is no conflicts.. :P  forgot to rebase

**Pull request merged**